docs/guides/cloud/account-management/billing-and-usage.mdx

```diff
@@ -21,7 +21,7 @@ sidebar_position: 50
 ## Open Source Plan
 
 To support the community, we provide the Open Source (OSS) plan to take
```

Comment:
```suggestion
To support the community, we provide the Open Source Software (OSS) plan to take
```

To match the acronym `OSS`, with its three letters, the word Software needs to be added.

Can also be written "Open-source software" according to https://en.wikipedia.org/wiki/Open-source_software